### DISCUSSION TITLE
About the resource option.

### DISCUSSION THREAD

**Discussion by thiagodapalmaoliveira** [xmppjs/xmpp.js#969] [Q&A] [answered]
How do I trigger a message without informing the resource? If the user is connected to two or more resources, is it possible to send a message to him in both sessions without having to inform the resource? Example:

```
const to = "19989989221@gingo.m.in-app.io";

const message = String("Hello world!").trim();
const body = xml("body", {}, message);

const stanza = xml(
	"message",
	{ type: "chat", to: jid(to) },
	body
);

await xmpp.send(stanza);
```

Every time I try this the message does not reach the user in any of the sessions. Can you help me?

Thank you very much and sorry for my horrible english!!

**Comment by sonnyp** [comment, marked as answer]
> Every time I try this the message does not reach the user in any of the sessions. Can you help me?

You have to send your presence.

`await xmpp.send(xml('presence'));`

**Reply by thiagodapalmaoliveira** [reply]
Thank you very much!!
